extensions/amp-video/0.1/video-cache.js

```diff
@@ -215,3 +234,18 @@ function getCacheUrlService(videoEl, ampdoc) {
     .installExtensionForDoc(ampdoc, 'amp-cache-url')
     .then(() => Services.cacheUrlServicePromiseForDoc(videoEl));
 }
+
+/**
+ * Returns `true` if the video's inline response should be used instead of
+ * issuing an XHR request.
+ * @param {!Element} videoEl
+ * @return {boolean}
+ */
+function shouldUseInlineVideoResponse(videoEl) {
+  // Monti only inlines the first video of the first web story page.
+  const isFirstVideoOnFirstStoryPage = matches(
+    videoEl,
+    `amp-story-page:first-of-type amp-video:first-of-type`
+  );
+  return isFirstVideoOnFirstStoryPage;
```

Comment:
```suggestion
  return matches(
    videoEl,
    `amp-story-page:first-of-type amp-video:first-of-type`
  );
```

Reply:
Just curious, did you keep the extra variable to be descriptive about what the `matches()` does?

Reply:
Yup, I only kept the extra variable for a little extra clarity